viz/RobotModel.cpp

```diff
@@ -23,6 +23,15 @@
 #include <fstream>
 #include <streambuf>
 
+#include <osgDB/Registry>
+#include <osgDB/ReadFile>
+#include <osgDB/WriteFile>
+#include <osgDB/FileNameUtils>
+#include <osgDB/ReaderWriter>
+#include <osgDB/PluginQuery>
+
+#define CALL_MEMBER_POINTER_FUNCTION(obj, fnc) ((obj).*(fnc))
```

Comment:
Really. Really. Not.

You use it ONCE and *even* if you don't like the syntax of call to pointer-to-member, this is the syntax every C++ developer should *know*.